pxc-80/Dockerfile

```diff
@@ -58,29 +58,33 @@ RUN groupadd -g 1001 mysql \
 RUN sed -i '/nodocs/d' /etc/yum.conf || : \
         && yum install -y \
                 yum-utils \
-        && curl -L -o /tmp/percona-xtradb-cluster-server.rpm https://repo.percona.com/pxc-80/yum/experimental/7/RPMS/x86_64/percona-xtradb-cluster-server-8.0.16-27dev.4.2.1.el7.x86_64.rpm \
-        && curl -L -o /tmp/percona-xtradb-cluster-shared.rpm  https://repo.percona.com/pxc-80/yum/experimental/7/RPMS/x86_64/percona-xtradb-cluster-shared-8.0.16-27dev.4.2.1.el7.x86_64.rpm \
-        && curl -L -o /tmp/percona-xtradb-cluster-client.rpm https://repo.percona.com/pxc-80/yum/experimental/7/RPMS/x86_64/percona-xtradb-cluster-client-8.0.16-27dev.4.2.1.el7.x86_64.rpm \
-        && curl -L -o /tmp/percona-xtradb-cluster-shared-compat.rpm https://repo.percona.com/pxc-80/yum/experimental/7/RPMS/x86_64/percona-xtradb-cluster-shared-compat-8.0.16-27dev.4.2.1.el7.x86_64.rpm \
+        && curl -L -o /tmp/percona-xtradb-cluster-server.rpm https://repo.percona.com/pxc-80/yum/experimental/7/RPMS/x86_64/percona-xtradb-cluster-server-8.0.18-26.4.3.1.el7.x86_64.rpm \
```

Comment:
Just as a suggestion:
add environment variable PXC_VERSION=8.0.18-26.4.3.1
in curl call use the following
curl -L -o /tmp/percona-xtradb-cluster-server.rpm https://repo.percona.com/pxc-80/yum/experimental/7/RPMS/x86_64/percona-xtradb-cluster-server-${PXC_VERSION}.x86_64.rpm \

so in future we could change version only in one line.